docs/en/advanced_guides/how_to.md

```diff
@@ -312,6 +312,45 @@ model = dict(
 )
 ```
 
+#### Don't used pre-training weights
+
+When we replace the backbone network, the model initialization is trained by default loading the pre-training weight of the backbone network. If, instead of using the pre-training weights of the backbone network, you want to train the time model from scratch,
```

Comment:
```suggestion
When we replace the backbone network, the model initialization is trained by default loading the pre-training weight of the backbone network. Instead of using the pre-training weights of the backbone network, if you want to train the time model from scratch,
```